README.md

```diff
@@ -87,7 +87,16 @@ hipsolverSgetrf(hipsolverHandle_t handle,
                 int*              devInfo);
 ```
 
-## Supported Fuctionality
+## Special Considerations with the rocSOLVER Backend
+Due to differences in implementation and API design between rocSOLVER and cuSOLVER, there are special considerations that should be taken into account when using hipSOLVER with the rocSOLVER backend.
+
+While many hipSOLVER functions (modeled after cuSOLVER functions) take a workspace pointer and size as arguments, rocSOLVER maintains its own internal device workspace by default. In order to take advantage of this feature, users may pass a null pointer for the `work` argument of any function when using the rocSOLVER backend, and the workspace will be automatically managed behind-the-scenes.
+
+Note that several functions - namely gesvd, getrs, and potrfBatched - will always use rocSOLVER's internal device workspace management. This may cause performance issues if combined with function calls that receive non-null `work` pointers, as the internal workspace will flip-flop between the user-provided and automatically allocated workspaces. It is recommended for programs that call gesvd, getrs, or potrfBatched to pass null pointers to the `work` arguments of all other functions.
```

Comment:
A single hypen is too small for an em dash. You can use Unicode:

> Note that several functions—namely gesvd, getrs, and potrfBatched—will always use rocSOLVER's

Or a few hyphens, if you want to stick to ASCII:
> Note that several functions---namely gesvd, getrs, and potrfBatched---will always use rocSOLVER's

FWIW, according to the Chicago and Harvard style manuals, there should be no spaces around the em dash. Adding spaces is more common in Canadian / UK style, but the ROCm docs follow US English style guides. (That said, even US authors often do this differently, so it's not that big of a deal.)